tc/sgx/trusted_worker_manager/enclave/enclave.edl

```diff
@@ -17,12 +17,6 @@ enclave {
     from "sgx_tstdc.edl" import *;
     from "sgx_tsgxssl.edl" import *;
     from "base.edl" import *;
-    from "signup.edl" import *;
-    from "signup_kme.edl" import *;
-    from "signup_wpe.edl" import *;
-    from "work_order.edl" import *;
```

Comment:
Rename `work_order.edl` to `work_order_singleton.edl` if this is specific to singleton enclave

Reply:
Done